<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonApply">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMusicSaveDir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Music Download Directory:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMusicSaveDir">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoDownload">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Download</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoHide">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Hide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOkSelect">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShowGuidaceOnStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Guidance When Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxQuality">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>High quality First</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Standard quality First</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Smooth quality First</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelQuality">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Quality:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOkOpen">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
